asn1tools/codecs/ber.py

```diff
@@ -297,7 +297,7 @@ def decode_real_special(control):
 
 
 def decode_real_decimal(data):
-    return float(data[1:])
+    return float(data[1:].replace(b'\x2c', b'\x2e'))
```

Comment:
Replace with ``return float(data[1:].replace(b',', b'.'))`` as it's easier to understand.